src/io/flutter/inspector/WidgetPerfPanel.java

```diff
@@ -102,6 +102,12 @@ public void selectionChanged(@NotNull FileEditorManagerEvent event) {
     Disposer.register(parentDisposable, perfTipComputeDelayTimer::stop);
   }
 
+  private static void handleTipSelection(@NotNull PerfTip tip) {
+    // Send analytics.
+    FlutterInitializer.getAnalytics().sendEvent("perf", "perfTipSelected");
```

Comment:
URL would be cumbersome since they're so long.  Could we consider adding a unique name to `PerfTipRule` that we could append to the event analytics?

So, something like:

    FlutterInitializer.getAnalytics().sendEvent("perf", "perfTipSelected." + tip.uniqueId);

?